assets/app/features/upgrade/i18n/en.json

```diff
@@ -76,6 +76,7 @@
             },
             "database-configuration": {
                 "title": "Enter Credentials and/or Information to create or connect to a PostgreSQL service to be used for serving the Crowbar database",
+                "title-completed": "This step has been completed. Click on button labeled \"Next\" to continue the upgrade process.",
```

Comment:
Could we just have *Click on "Next" button...* here? Unless there's some policy for this wording...